builtin/providers/aws/resource_aws_elasticache_cluster_test.go

```diff
@@ -23,6 +23,8 @@ func TestAccAWSElasticacheCluster(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckAWSElasticacheSecurityGroupExists("aws_elasticache_security_group.bar"),
 					testAccCheckAWSElasticacheClusterExists("aws_elasticache_cluster.bar"),
+					resource.TestCheckResourceAttr(
+						"aws_elasticache_cluster.bar", "cache_nodes.0.id", "001"),
```

Comment:
Did this pass for you? My original TODO when I kicked it over was this failing because the nodes don't show up right away, so I figured we had to add a `retryFunc` to poll until the nodes show up in the response.